a/amazon-metadata.yml

```diff
@@ -0,0 +1,19 @@
+amazon-metadata:
+  image: rancher/os-amazon-metadata:v0.9.2${SUFFIX}
```

Comment:
yeah, `build-images` is dumb - but I only push images that actually have changes that are worth pushing.